Corona-Warn-App/src/main/res/layout/fragment_submission_dispatcher.xml

```diff
@@ -107,7 +107,7 @@
                     layout="@layout/include_dispatcher_card"
                     android:layout_width="0dp"
                     android:layout_height="wrap_content"
-                    android:layout_marginTop="@dimen/spacing_normal"
+                    android:layout_marginTop="16dp"
```

Comment:
In the design there should not be any margins for in the images it is aligned to bottom right corner
<img width="540" alt="Screenshot 2021-05-19 at 13 01 06" src="https://user-images.githubusercontent.com/25054729/118802347-4ebaa680-b8a2-11eb-91e0-27311070b5b0.png">

Reply:
is now fixed